src/ui/Header.tsx

```diff
@@ -43,21 +43,7 @@ export default function Header(
   const mainBgColor = useColorModeValue('ui.white', 'ui.black', 'ui.sepia');
 
   return (
-    <Flex
-      as="header"
-      position="sticky"
-      top={0}
-      left={0}
-      right={0}
-      height={`${HEADER_HEIGHT}px`}
-      zIndex="sticky"
-      alignContent="space-between"
-      alignItems="center"
-      px={8}
-      borderBottom="1px solid"
-      borderColor="gray.100"
-      bgColor={mainBgColor}
-    >
+    <HeaderWrapper bgColor={mainBgColor}>
```

Comment:
`HeaderWrapper` component is missing this bgColor style.  I think this is needed or else you will get a weird transition bg color.